gym/vector/vector_env.py

```diff
@@ -49,129 +48,112 @@ def __init__(self, num_envs, observation_space, action_space):
 
     def reset_async(
         self,
-        seed: Optional[Union[int, List[int]]] = None,
+        seed: Optional[Union[int, list[int]]] = None,
         return_info: bool = False,
         options: Optional[dict] = None,
     ):
+        """Reset the vector environments asynchronously."""
         pass
 
     def reset_wait(
         self,
-        seed: Optional[Union[int, List[int]]] = None,
+        seed: Optional[Union[int, list[int]]] = None,
         return_info: bool = False,
         options: Optional[dict] = None,
     ):
+        """Resets the vector environments while waiting for the results."""
```

Comment:
The resets are usually done in `reset_async`

```suggestion
        """Retrieves the results of a :meth:`reset_async` call.
        
        A call to this method must always be preceded by a call to :meth:`reset_async`.
        """
```